src/Form/Type/EveryPayConfigurationType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdeoWeb\SyliusEveryPayPlugin\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+final class EveryPayConfigurationType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('api_url', UrlType::class, [
+                'label' => 'adeoweb_sylius_everypay_plugin.configuration.api_url',
+                'required' => true,
+                'constraints' => [
```

Comment:
nepamenu dabar, kaip tiksliai šita vieta ant syliuso veikia, bet nebūtų geriau šitam form type uždėti data_class į EveryPayConfiguration ir tada konstraintus yamle ant tai klasei surašyti?

Reply:
Čia gali būti, kad payumo integracijos savybė, kad vaikšinėja masyvas, o ne objektas.

Manau galim šitą klausimą ateičiai pasilikti, jei ką.

Reply:
Pasirodo, ta klasė ne veltui dažniausiai vadinasi `*Api`. Ji yra pagrindinis API klientas, o ne konfigūracijos Data objektas. Tad ir validaciją prie jos kabinti turbūt nebūtų logiška.

Bet pridėjau pačios Api klasės konstruktoriuje assertą ir nurodžiau `GatewayFactory`, kad šitie laukai privalomi ir turi būti ne tušti. Manau to pakaks kol kas, nors tikrai sutinku, kad čia nelabai Symfony stiliuje gaunasi.